validation/src/utils/request/index.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+const axios = require('axios');
+const https = require('https');
+const io = require('socket.io-client');
+const { apiClient } = require('@liskhq/lisk-client');
+const config = require('../../../config');
+
+const { getCertificateFromUrl, convertCertificateToPemPublicKey } = require('./certificate');
+
+const agent = new https.Agent({
+	rejectUnauthorized: true,
+});
+
+const validatePublicKeyFromURL = async (url, publicKey) => {
+	const sslCertificate = await getCertificateFromUrl(url);
```

Comment:
Just to be consistent, either we can update the method name to `getCertificateFromURL` or `validatePublicKeyFromUrl`
```suggestion
const validatePublicKeyFromURL = async (url, publicKey) => {
	const sslCertificate = await getCertificateFromUrl(url);
```